pandas/tests/io/test_sql.py

```diff
@@ -1596,13 +1514,14 @@ def test_api_to_sql_index_label(conn, request, index_name, index_label, expected
     assert frame.columns[0] == expected
 
 
-@pytest.mark.db
 @pytest.mark.parametrize("conn", all_connectable)
 def test_api_to_sql_index_label_multiindex(conn, request):
     conn_name = conn
     if "mysql" in conn_name:
         request.node.add_marker(
-            pytest.mark.xfail(reason="MySQL can fail using TEXT without length as key")
+            pytest.mark.xfail(
+                reason="MySQL can fail using TEXT without length as key", strict=False
```

Comment:
Was this still flaky (is `strict=False`) still needed?

Reply:
Yea I think it depends on the version of MySQL and may even be a MySQL versus MariaDB difference. I couldn't pinpoint exactly where this would have been allowed

https://stackoverflow.com/questions/1827063/mysql-error-key-specification-without-a-key-length